### PR TITLE
Add homepage and about page

### PR DESCRIPTION
**Pull request by kwekubright**
### Description:
In this feature I have achieved the following:

- Added Homepage with speakers and partner section
- Added About page with event details and sections to preview the previous events

Here is a link to my loom video [LOOM](https://www.loom.com/share/d065eea3ab2a4547b8573aa28e8b44e1)

html-capstone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+-  Cindy Shin on Behance
+- Flaticon.com
+- Pexels.com
+- Freepik.com
+- Fontawesome.com
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (MahmoudBakr23) [inline README.md:59]: **Optional**
- Please fix the MIT link for a completely professional repo. 🙏

File: about.html [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+            <h2 class="text-black section-title">About DevCAMP</h2>
+          </div>
+          <p class="lead mb-5 text-center">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quidem.
+            Quasi, quisquam. Quasi, quisquam. Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quidem.
+            Quasi, quisquam. Quasi, quisquam. Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quidem.
+            Quasi, quisquam. Quasi, quisquam. Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          </p>
+        </div>
+        <div class="col-lg-12 d-flex justify-content-center align-items-center flex-column">
+          <h3 class="call-to-action">Let's Code!</h3>
```
Comment (MahmoudBakr23) [inline about.html:103]: **Highlights**
- Great job styling this element. 😉

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @kwekubright 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Great design.
- Well done making your app fully responsive using the CSS media queries.
- Correct Linters
- Deployed
- Descriptive README and PR
- Semantic HTML
- Great Presentation

**Optional**
_Please see the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 2 above)
Comment (kwekubright) [reply about.html:103]: Thank you @MahmoudBakr23

**Pull request merged**